### PR TITLE
Updated Windows Wasmer icons. Add wax

### PR DESCRIPTION
**Pull request by syrusakbary**
# Description

Updated Wasmer icons. Updated wax

# Review

- [x] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -287,6 +287,8 @@ build-install-package:
 	mkdir -p ./install/bin
 	cp ./wapm-cli/target/release/wapm ./install/bin/
 	cp ./target/release/wasmer ./install/bin/
+	# Create the wax binary as symlink to wapm
+	cd ./install/bin/ && ln -s wapm wax
 	tar -C ./install -zcvf wasmer.tar.gz bin/wapm bin/wasmer
```
Comment (lmorchard) [inline Makefile]: Swinging by to check out wax and finding it missing in the release tarballs. Should `bin/wax` be added to the list here?

If I manually run `ln -s $HOME/.wasmer/bin/wapm $HOME/.wasmer/bin/wax`, the wax command works for me locally after installing via curl

Comment (MarkMcCaskey) [reply Makefile]: Thanks for letting us know!  I'm looking into this now

Comment (MarkMcCaskey) [reply Makefile]: We had a retro on our release process and will be making some changes to reduce the chance of something like this happening again.  Thanks again for reporting this and verifying that it was working on your end after we shipped the fix!

Comment (syrusakbary) [reply Makefile]: Here's the link to our retro where we analyzed the issues during the release process:

https://www.notion.so/wasmer/March-12-Release-Retrospective-2a5d1b79005f43fe9ee6a035cf0b5921

Thanks for the ping @lmorchard!

**Pull request merged**